notebooks/en/llm_gateway_pii_detection.ipynb

```diff
@@ -0,0 +1,536 @@
+{
```

Comment:
<p>I would spell out what PII stands for the first time you use it, and then you can use the acronym in the rest of the recipe.</p><p> </p><p>"...scrubbing of Personal Identifiable Information (PII) data to mitigate the risk of PII leaking."</p><p> </p><p>Maybe also add a link for users to sign up for a free trial API.</p><p> </p><p>"...free trial API for Cohere's..."</p><p> </p><p>If you can, I think it'd be a better user experience if you copy the install instructions into this notebook so users don't have to switch back and forth.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Addressed!</p>